tasks/security_updates.yml

```diff
@@ -1,11 +1,73 @@
 ---
 
-- name: Ensures latest Security updates are installed
-  become: yes
-  shell: "{{ item }}"
-  with_items:
-    - echo "# Running security udpate (`date`)" >> /var/log/apt-security-updates
-    - apt-get update >> /var/log/apt-security-updates
-    - echo "\n# Installing updates" >> /var/log/apt-security-updates
-    - DEBIAN_FRONTEND=noninteractive apt-get upgrade -o Aptitude::Delete-Unused=false --assume-yes --target-release `lsb_release -cs`-security >> /var/log/apt-security-updates
-    - echo "Security updates (if any) installed\n\n\n\n" >> /var/log/apt-security-updates
+- name: Install aptitude
+  become: yes
+  apt:
+    cache_valid_time: 3600
+    name: aptitude
```

Comment:
Why is Aptitude still needed? Haven't we removed the need to this by switching to the apt module?

Reply:
The apt module uses aptitude as the backend; a fallback to "pure" apt is available in Ansible 2.4+.